pages/story/story_data.js

```diff
@@ -0,0 +1,28 @@
+
+//write functions here to take data and push to story_array
+
+export const story_array = 
+[{
+    num : 1,
+    href: "/story/1", //keep this unique
+    teampic: "https://images.unsplash.com/photo-1628891890467-b79f2c8ba9dc?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Mnx8c3BvcnRzJTIwdGVhbXxlbnwwfHwwfHw%3D",
+    profilepic: "https://images.unsplash.com/photo-1527980965255-d3b416303d12?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8NXx8YXZhdGFyfGVufDB8fDB8fA%3D%3D",
+    name: "Ramkumar V.",
+    date: "2022/23",
+    title: "The one where we beat Goa :)",
+    brief_content: "Here are the biggest achievements, fun and some nerve cracking moments of last year!",
```

Comment:
camel case here too. Also brief and body contact seem to be same. can we think of better naming otherwise seems confusing and have to look in code to find the difference

Reply:
changed it to `intro` and `body`